### DISCUSSION TITLE
Quadmesh data Support (xCDAT)

### DISCUSSION THREAD

**Discussion by gkb999** [xCDAT/xcdat#478] [2. Questions]
Hi, I just wanted to know if xcdat is capable of handling the 'quadmesh' type of data.
Reason, the same temporal climatology and departure functions that work on ERA5 data, doesn't seem to be working on the "quademsh" data (although in netcdf version.

Is there something (Literature/Documentation) on this that I'm misisng?
Please assist.

My Data looks like:
```python
<xarray.Dataset>
Dimensions:              (time: 30, x: 1264, y: 1328)
Coordinates:
  * x                    (x) float64 -3.947e+06 -3.941e+06 ... 3.947e+06
  * y                    (y) float64 -3.947e+06 -3.941e+06 ... 4.347e+06
Dimensions without coordinates: time
Data variables:
    polar_stereographic  (time) |S1 b'' b'' b'' b'' b'' ... b'' b'' b'' b'' b''
    z                    (time, y, x) float32 dask.array<chunksize=(1, 1328, 1264), meta=np.ndarray>
Attributes:
    author:       University of Bremen, Gunnar Spreen [gunnar.spreen@uni-brem...
    Conventions:  CF-1.5
    GMT_version:  5.2.1 (r15220) [64-bit] [MP]
    history:      Mon Jan 21 20:08:59 2019: GDAL CreateCopy( /ssmi/www/htdocs...
    info:         AMSR2 sea ice concentration based on the ASI algorithm (Spr...
    title:        Produced by grdmath
```

**Comment by jypeter** [comment]
I very probably will not be able to answer your question, but can you **be more specific** than _doesn't seem to be working_ if you need an answer? We don't know the exact python command that generated the error, and we don't know the error either

It would also help to know the exact meta-data of the nc file you are working with. On a Linux computer, I would use `ncdump -h file.nc` to display the header of the file, but I don't know if there is a `ncdump` available for Windows. Do you have access to Linux servers for your work? It would probably be easier

**Reply by tomvothecoder** [reply]
Thanks for the comments @jypeter!, I agree with your points!

Just quoting myself from another [post](https://github.com/xCDAT/xcdat/discussions/476#discussioncomment-5930554) about how to format questions:

> Hi @gkb999, please follow this new [template](https://github.com/xCDAT/xcdat/discussions/new?category=2-questions) for asking questions in this forum. It really helps get things moving if questions and comments are succinct. Open-source projects (including ours) usually require users provide a [minimum, reproducible code example](https://stackoverflow.com/help/minimal-reproducible-example) with the relevant stack trace in proper markdown formatting.
> 
> If you want, you can transfer the content relevant from this post over to the new post.

**Comment by tomvothecoder** [comment]
Closing this discussion for a new discussion post using questions template
